### PR TITLE
Add vue3 code-sample

### PR DESCRIPTION
**Pull request by bidoubiwa**
Vue3 code-sample was asked for and missing in the front-end integration samples. This PR add its.

⚠️ DO NOT MERGE before this PR is merged: https://github.com/meilisearch/meilisearch-vue/pull/121

### CODE REVIEW

**Comment by guimachiavelli** [comment]
So, what's the status on this PR? Is anything blocking it? Is it still relevant?

**Comment by bidoubiwa** [comment]
Completely forgot about this. I'm adding this to my todo! I'll keep you updated

**Comment by bidoubiwa** [comment]
Can you review the PR ?

File: .vuepress/code-samples/sdks.json
Comment (bidoubiwa) [inline .vuepress/code-samples/sdks.json:89]:
```suggestion
    "cacheableTab": false,
    "ignoreInSamplesReport": true,
```

**Comment by guimachiavelli** [comment]
@bidoubiwa, @dichotommy has approved the linked PR. Are we missing anything else? If not, can you mark it as ready for review? I'd be happy to take a final look and approve it if we are good to go.

**Comment by bidoubiwa** [comment]
Hey @guimachiavelli 
I was waiting for someone with enough rights to approve on this repository. @CaroFG came to the rescue :) I'm merging now

**Comment by bidoubiwa** [comment]
You can merge this!

**Review by guimachiavelli — APPROVED** [review]
🐶

**Comment by guimachiavelli** [comment]
bors merge

**Pull request merged**
